script.js

```diff
@@ -70,12 +70,13 @@ async function rstToHtml() {
 
     outputFrame.srcdoc = result;
 
-    // Override Docutils' default style, which adds a grey background to the body element.
+    // Apply consistent styling to the iframe content using our docutils-specific CSS.
     // We need to wait until the iframe's load event; see https://stackoverflow.com/a/13959836/266309.
     outputFrame.addEventListener("load", (event) => {
-      const newStyle = outputFrame.contentDocument.createElement("style");
-      newStyle.textContent = "body { background-color: unset; }";
-      event.target.contentDocument.head.appendChild(newStyle);
+      const link = outputFrame.contentDocument.createElement("link");
```

Comment:
"Link" is too generic a name, let's use something more explicit:

```suggestion
      const linkTag = outputFrame.contentDocument.createElement("link");
```